frappe/public/scss/common/css_variables.scss

```diff
@@ -261,8 +261,13 @@ $input-height: calc(1.5em + 0.75rem);
 	--awesomplete-hover-bg: var(--gray-150);
 
 	// Button Colors
-	--btn-default-bg: var(--gray-100);
-	--btn-default-hover-bg: var(--gray-300);
+	--btn-default-bg: white;
+	--btn-default-hover-bg: var(--gray-100);
+	--btn-default-active-bg: var(--gray-100);
```

Comment:
`btn-default-active-bg` ... we'll use `btn-default-hover-bg` for active styling
`btn-default-hover-bg` is being duplicated here